source/server/README.md

```diff
@@ -114,6 +115,9 @@ bazel-bin/nighthawk_test_server  [--use-fake-symbol-table <bool>]
 
 Where:
 
+--disable-extensions <string>
```

Comment:
Are these changes related to this PR?

Reply:
Yeah, this is auto generated and originates from the updated Envoy version which apparently has a new cli arg. As ‘nighthawk_test_server’ basically is Envoy + a static extension we may see this when updating the dependency.